tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/ServerlessMiniAgent.cs

```diff
@@ -0,0 +1,113 @@
+// <copyright file="ServerlessMiniAgent.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.IO;
+
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.ServerlessInstrumentation;
+
+internal class ServerlessMiniAgent
+{
+    private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(ServerlessMiniAgent));
+    internal const string AzureFunctionNameEnvVar = "WEBSITE_SITE_NAME";
```

Comment:
Some of those are already defined in https://github.com/DataDog/dd-trace-dotnet/blob/cd9b0807887d1c09cc371934845627a36ff2af99/tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.AzureAppService.cs#L38. I suggest we put all the Azure one in the file I linked.  We could create an equivalent for GCP.

Reply:
ah right, good idea. I added all the azure env var keys into the AppService config file, and created a new GCPFunction one with all the GCP function keys.